### ISSUE TITLE
Adding support for Incus

### ISSUE THREAD

**Issue by bds-congnguyen** [LINBIT/linstor-server#383]
Hi there!
[Incus ](https://linuxcontainers.org/incus/) is the new linux container, the next generation of system container and virtual machine manager, the folk of LXD.

Currently, I didn't see Linstor support Incus as storage driver. so I'd like to ask for this feature supported by Linstor. 

Best regards.

**Comment by rck** [comment]
Just some quick notes from one that briefly looked into LXD support for LINSTOR a very long time ago: Should be doable, but we already have a ton of plugins to maintain and to be honest as long as there is no real market demand (i.e., paying LINBIT customers), I don't think this is going to happen from our side. FLOSS contributions obviously always welcome.

**Comment by candlerb** [comment]
Work is already underway at the incus side:
* https://github.com/lxc/incus/issues/564
* https://github.com/lxc/incus/pull/1621

I'm not sure that anything needs to be done on the linstor side, although there's something about "storage buckets" in [this comment](https://github.com/lxc/incus/pull/1621#issuecomment-2638181920)

**Comment by bensmrs** [comment]
Yeah, we’re working hard on the feature. If something can be done on LINSTOR side, we don’t hesitate to ask (see https://github.com/LINBIT/golinstor/issues/18). The real struggle is to get past the lack of renaming support, but we’re getting there :)

**Comment by rck** [comment]
@bensmrs 
> The real struggle is to get past the lack of renaming support, but we’re getting there

without knowing any details what you intend to rename, but yes, that problem exists on several levels. you might be aware that LINSTOR can store "aux properties" (arbitrary key/value strings). We use these in various places like storing which disk belongs to which VM. You could use that to store the user-friendly name that can change (i.e., just an update of the aux property) and use static names (usually a prefix + UUID) as actual static LINSTOR/DRBD names. HTH, feel free to ask us details on what you actually try to rename, the people that invented it might hove some idea here and there :)

**Comment by bensmrs** [comment]
That’s exactly what we’re doing :)
@luissimas has discussed about it on https://github.com/LINBIT/linstor-server/issues/44

**Comment by luissimas** [comment]
Hey folks!

The LINSTOR storage driver was merged and released with Incus 6.11 last week. For instructions on how to use it, check out the [docs](https://linuxcontainers.org/incus/docs/main/reference/storage_linstor/).

**Comment by bds-congnguyen** [comment]
Thank @luissimas , waiting so long to try it
